multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/v2/ServicesCloudModelBuilder.java

```diff
@@ -73,7 +71,7 @@ protected CloudServiceInstanceExtended createManagedService(Resource resource, C
                                                     .resourceName(resource.getName())
                                                     .label((String) parameters.get(SupportedParameters.SERVICE))
                                                     .plan((String) parameters.get(SupportedParameters.SERVICE_PLAN))
-                                                    .type(ServiceInstanceType.MANAGED)
+                                                    .type(org.cloudfoundry.client.v3.serviceinstances.ServiceInstanceType.MANAGED)
```

Comment:
why not just:
import org.cloudfoundry.client.v3.serviceinstances.ServiceInstanceType;